modules/client-configuration/pages/client-proxy-script.adoc

```diff
@@ -0,0 +1,69 @@
+[[script-client-proxy]]
+= Register Clients  to a Proxy with a Bootstrap Script
+
+
+You can register clients through the {productname} Proxy with a bootstrap script.
+This is done almost the same way as registering clients directly with the {productname} Server.
+
+You create the bootstrap script in the {productname} Server container with the [command]``mgr-bootstrap`` command line tool.
+[command]``mgr-bootstrap`` requires some parameters such as the fully qualified domain name (FQDN) of the proxy, activation keys, or GPG keys.
+These parameters depend on your specific setup, but the FQDN of the proxy is mandatory.
+
+The bootstrap script then deploys all necessary information to the clients.
+
+
+
+.Procedure: Registering Clients to a Proxy with a Bootstrap Script
+
+. Create a client activation key on the {productname} Server using the {webui}.
+  For more information, see xref:client-configuration:activation-keys.adoc[].
+. On the container host of the server, start a terminial in the container:
+
++
+
+[source,shell]
+----
+mgrctl term
+----
+
++
+
+--
+.. In the container, to create the bootstrap script, execute the [command]``mgr-bootstrap`` command, for example, with these options:
+
++
+
+[source,shell]
+----
+mgr-bootstrap --hostname=MLM_PROXY --activation-keys=ACTIVATION_KEY \
+  --script bootstrap-MLM_PROXY.sh
+----
+
++
+
+Replace [literal]``MLM_PROXY`` with the host name of your proxy.
```

Comment:
Not sure how exactly do this, but I think it would improve readability if there was a new line after this line. Is it like this?

```suggestion
Replace [literal]``MLM_PROXY`` with the host name of your proxy.

+
```
```

Reply:
yes, but additionally removing indentation of the following line is needed...